carma_globaer.h

```diff
@@ -43,15 +43,9 @@
 !  Gridding Information
 #define igridv        cstate%f_igridv
 #define igridh        cstate%f_igridh
-#define xmet          cstate%f_xmet
-#define ymet          cstate%f_ymet
 #define zmet          cstate%f_zmet
 #define zmetl         cstate%f_zmetl
-#define xc            cstate%f_xc
```

Comment:
May want topreserve xc,yc,dx,dy for the stand-alone code. For example using it in an LES type environment were cartesian coordinates are more natural to identify a location than lat/lon.